ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -34,6 +34,10 @@ def Atan(*, x, name="atan"):
     return ivy.atan(x)
 
 
+def Atan2(*, y, x, name="atan2"):
```

Comment:
well, the origenal function tf.raw_ops.Atan2 only has these arguments:
tf.raw_ops.Atan2(
    y, x, name=None
)
So your implementation should have the same arguments as well.

Reply:
Hi Daniel, I've changed it as per your comment, however, the other functions in the file follow the old format (with "*" and name=function_name)
Should I also change the other functions in the file to similar formatting?